### PR TITLE
[Merged by Bors] - Support model pruning and quantization

### LINKED ISSUE #3: Support model pruning and quantization

### PR DESCRIPTION
**Pull request by ChengcanWang-zte**
Close #3

### CODE REVIEW

File: ResNet-50-Traing-Pruning.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# ResNet-50 training and pruning
```
Comment (shiyingjin) [inline ResNet-50-Traing-Pruning.md]: Put this file into a separate directory

**Review by shiyingjin — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,4 +1,158 @@
...
+|   model   |   card    | batch size | learning-rate |
+| --------- | --------- | ---------- | ------------- |
+| ResNet-50 | V100 32GB | 256        | 0.1           |
+| ResNet-50 | P100 16GB | 128        | 0.05          |
+
+
+
 
 [![Build Status](https://dev.azure.com/Adlik/GitHub/_apis/build/status/Adlik.model_optimizer?branchName=master)](https://dev.azure.com/Adlik/GitHub/_build/latest?definitionId=2&branchName=master)
 [![Bors enabled](https://bors.tech/images/badge_small.svg)](https://app.bors.tech/repositories/23640)
```
Comment (EFanZh) [inline README.md]: Put these lines after the title.

File: docker/Dockerfile.cpu [hunk 3]
```diff
@@ -0,0 +1,61 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+        apt-get install -y software-properties-common && \
+        add-apt-repository ppa:deadsnakes/ppa && \
```
Comment (EFanZh) [inline docker/Dockerfile.cpu]: No need to use external Python repository. The Python from official Ubuntu 18.04 packages is 3.6.

File: docker/Dockerfile.cpu [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    echo "Port 10022" >> /etc/ssh/sshd_config.new && \
+    mv /etc/ssh/sshd_config.new /etc/ssh/sshd_config && \
+    ssh-keygen -t rsa -N '' -f ~/.ssh/id_rsa && \
+    sed -i -E -e 's/^PermitRootLogin\s+prohibit-password$/PermitRootLogin yes/' /etc/ssh/sshd_config && \
+    cat ~/.ssh/id_rsa.pub >> ~/.ssh/authorized_keys
+
+# Install Model Optimimzer
+RUN pip install  --no-cache-dir jsonschema==3.1.1 && \
+    pip install  --no-cache-dir networkx==2.4 && \
+    pip install  --no-cache-dir mpi4py==3.0.3 && \
```
Comment (EFanZh) [inline docker/Dockerfile.cpu]: Dependencies specified in `setup.py` will be automatically installed when the target package is being installed. So you don’t need to install them manually.

**Review by EFanZh — CHANGES_REQUESTED** [review]

**Review by shiyingjin — APPROVED** [review]

**Review by EFanZh — APPROVED** [review]

**Pull request closed without merging**